airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -149,6 +171,13 @@ def _metadata_upload(
     return upload_file_if_changed(metadata_file_path, bucket, latest_path, disable_cache=True)
 
 
+def _clear_release_candidate(bucket: storage.bucket.Bucket, metadata: ConnectorMetadataDefinitionV0) -> None:
+    release_candidate_path = get_metadata_remote_file_path(metadata.data.dockerRepository, RELEASE_CANDIDATE_GCS_FOLDER_NAME)
+    release_candidate_blob = bucket.blob(release_candidate_path)
+    if release_candidate_blob.exists():
```

Comment:
Can we log if it doesn't exist? I think if we hit this line we expect it to be there.

Reply:
This function is not actually called, I just removed it, the full logic is in `delete_release_candidate_from_gcs`. We do raise errors on missing expected files there.